### PR TITLE
speaker/speech mapping accuracy and gold standard file

### PR DESCRIPTION
**Pull request by liamtabib**
This PR includes the script for generating the accuracy for the mapping of speakers to speeches, into /scripts. Moreover, it also includes the combined gold standard files from the three annotators, into /input/gold_standard.

### CODE REVIEW

**Comment by BobBorges** [comment]
@liamtabib Could you explain what you're doing with scipy.stats.beta.ppf?

**Comment by liamtabib** [comment]
I don't know! I based it off a similar script from Väinö. So could you explain @ninpnin

**Comment by MansMeg** [comment]
I think it is just the Bayesian posterior in a binomial experiment. But @ninpnin should know.

**Comment by ninpnin** [comment]
Yeah, you formulate a binomial model and calculate the 90% (or 95%) credible interval. Maybe we should add that as a comment?

File: scripts/mps_mapping_accuracy_estimate.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    byyear_sum.to_csv("results.csv", index=False)
+
+
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument("--start", type=int, default=1867)
+    parser.add_argument("--end", type=int, default=2022)
+    parser.add_argument("--path_goldstandard", type=str, default='PATH/TO/GOLDSTANDARD')
```
Comment (BobBorges) [inline scripts/mps_mapping_accuracy_estimate.py]: It's a small thing, but why set a dummy default path? 
Either set the path to a files o that the script won't fail if `--path_goldstandard` isn't set or replace `default='PATH/TO...'` with `required=True` so it must be set.

File: scripts/mps_mapping_accuracy_estimate.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    byyear = df['accuracy'].groupby(df['decade'])
+    byyear_sum = byyear_sum.merge(byyear.mean(), on="decade").reset_index()
+    print(byyear_sum)
+    byyear_sum.to_csv("results.csv", index=False)
+
+
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description=__doc__)
```
Comment (BobBorges) [inline scripts/mps_mapping_accuracy_estimate.py:95]: Set a doc string.

**Review by BobBorges — APPROVED** [review]

**Comment by MansMeg** [comment]
@liamtabib Can you fix Bobs issues?

**Comment by liamtabib** [comment]
Docstring has been added and path argument improved. If there are no further improvements, we can merge.

**Comment by MansMeg** [comment]
Great! I would like @ninpnin to take a pass before merge, then Im happy.

**Comment by ninpnin** [comment]
LGTM

**Pull request merged**